generate/param_generator.go

```diff
@@ -102,14 +103,16 @@ func GenParamEnum() {
 			fallthrough
 		case "string":
 			stringParamMap[name] = rawName
+		case "stringSlice":
+			stringSliceParamMap[name] = rawName
 		case "int":
 			intParamMap[name] = rawName
 		case "bool":
 			boolParamMap[name] = rawName
 		case "duration":
 			durationParamMap[name] = rawName
 		default:
-			errMsg := "UnknownType, add a new struct and return method to the generator or add it to one of the already handles types"
+			errMsg := fmt.Sprintf("UnknownType '%s', add a new struct and return method to the generator or add it to one of the already handles types", pType)
```

Comment:
Noticed a grammar error, probably mine originally.  Change `handles` to `handled`